src/Disciplina/Launcher/Resource.hs

```diff
@@ -0,0 +1,63 @@
+
+-- | Common resources used by Disciplina nodes
+
+module Disciplina.Launcher.Resource
+       ( BasicNodeResources (..)
+       , bracketBasicNodeResources
+       ) where
+
+import Universum
+
+import Mockable (Production)
+import System.Wlog (LoggerConfig (..), LoggerName, WithLogger, maybeLogsDirB, parseLoggerConfig,
+                    productionB, removeAllHandlers, setupLogging, showTidB)
+
+import Disciplina.Launcher.Params (BasicNodeParams (..), LoggingParams (..))
+
+-- | Datatype which contains resources required by all Disciplina nodes
+-- to start working.
+data BasicNodeResources = BasicNodeResources
+    { bnrLoggerName :: !LoggerName
+    }
+
+----------------------------------------------------------------------------
+-- Logging
+----------------------------------------------------------------------------
+
+readLoggerConfig :: MonadIO m => Maybe FilePath -> m LoggerConfig
```

Comment:
Why use mtl-style MonadIO instead of concrete Production type with these logging functions, readLoggerConfig, getRealLoggerConfig and setupLoggers?

Reply:
Just a habit, I guess. Also, it might be so that in the future logging initialization isn't done in `Production` monad (e. g. in Cardano it's done directly in IO monad for reasons of some advanced initial logging, which we don't want yet)

Reply:
Ok, makes sense.